lib/rubocop/cop/sorbet/binding_constants_without_type_alias.rb

```diff
@@ -58,6 +67,14 @@ def not_nil?(node)
 
         def on_casgn(node)
           return unless binding_unaliased_type?(node) && !using_type_alias?(node.children[2])
+          if using_deprecated_type_alias_syntax?(node.children[2])
+            add_offense(
+              node.children[2],
+              message: "It looks like you're using the old `T.type_alias` syntax. " \
+              '`T.type_alias` now expects a block.'
```

Comment:
Should we add a note here on how to fix this? For example:
`Run Sorbet with the options "--autocorrect --error-white-list=5043" to automatically upgrade to the new syntax.`

Reply:
^ I like this, pushed a new version with the error message.